README.md

```diff
@@ -1,59 +1,49 @@
 ![](https://img.shields.io/badge/Microverse-blueviolet)
 
-# Project Name
+# CoronaWide
 
-> One paragraph statement about the project.
+"CoronaWide" is a website that display the world statistics for covid-19.
 
-![screenshot](./app_screenshot.png)
+![screenshot](./src/assets/homepage.png)
+![screenshot](./src/assets/Details.png)
 
-Additional description about the project and its features.
+It is a Single Page App (SPA) that allows users to:
+    -> Search country to get up to date data about covid 19.
+    -> Every country as is own further details
+    
+## Project presentation
+
+[Presentation video](https://www.loom.com/share/947363dba4b94a3c92f74da4bcb82598)
 
 ## Built With
 
-- Major languages
-- Frameworks
-- Technologies used
+- HTML5, CSS3
+- JavaScript
+- React
+- Redux
 
 ## Live Demo
 
-[Live Demo Link](https://livedemo.com)
-
+[Netlify](https://coronawide.netlify.app/)
 
 ## Getting Started
 
-**This is an example of how you may give instructions on setting up your project locally.**
-**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
-
-
 To get a local copy up and running follow these simple example steps.
 
-### Prerequisites
-
 ### Setup
 
-### Install
-
-### Usage
-
-### Run tests
```

Comment:
- [ ] Good job adding the unit tests for the components. However, according to the project requirements below, you are missing integration tests using the **React Testing Library**. Kindly add them.

![image](https://user-images.githubusercontent.com/36542986/142614747-9ed46e73-7479-4f23-9e37-0ccf68396182.png)